build/package/migrate.Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Base Go environment
+# -------------------
+FROM golang:1.21-alpine as base
+WORKDIR /hatchet
+
+RUN apk update && apk add --no-cache gcc musl-dev git
+
+COPY go.mod go.sum ./
+COPY /prisma ./prisma
+
+RUN go install github.com/steebchen/prisma-client-go@v0.31.2
```

Comment:
@abelanger5 We should just do `go mod download`, otherwise this version will get out of sync with the one in `go.mod`.